src/main/java/io/supertokens/webserver/api/multitenancy/thirdparty/CreateOrUpdateThirdPartyConfigAPI.java

```diff
@@ -61,6 +65,11 @@ protected void doPut(HttpServletRequest req, HttpServletResponse resp) throws IO
         thirdPartyId = thirdPartyId.trim();
 
         try {
+            if (Arrays.stream(FeatureFlag.getInstance(main, new AppIdentifier(null, null)).getEnabledFeatures())
```

Comment:
this should only be done if not doing for null, null, null tenant

Reply:
Done